myuw/data/resource_link_import.csv

```diff
@@ -25,17 +25,19 @@ Academics,Tools and Software,all,http://canvas.uw.edu/,Canvas LMS,,,,,,,no
 Academics,Tools and Software,all,,,http://scout.uw.edu/seattle,Scout: Discover the UW,http://scout.uw.edu/bothell,Scout: Discover the UW,http://scout.uw.edu/tacoma,Scout: Discover the UW,no
+Academics,Tools and Software,all,http://grad.uw.edu/for-students-and-post-docs/mygrad-program/,MyGrad – Graduate School Portal,,,,,,,
 Academics,Tools and Software,all,http://www.washington.edu/itconnect/wares/uware/tableau-software/,Tableau Data Analysis Software,,,,,,,no
 Academics,Tools and Software,all,https://panopto.uw.edu/,Panopto Lecture Capture,,,,,,,no
 Academics,Tools and Software,all,https://itconnect.uw.edu/wares/,Other Software and Hardware Services,,,,,,,
-"Accounts, Identity, and Email",Email,all,http://alpine.washington.edu/,Alpine Email,,,,,,,no
-"Accounts, Identity, and Email",Email,all,https://outlook.office365.com/uw.edu,UW Office 365 Email,,,,,,,no
-"Accounts, Identity, and Email",Email,all,http://mail.google.com/a/uw.edu,UW Gmail,,,,,,,no
-"Accounts, Identity, and Email",Email,all,http://outlook.com/owa/myuw.net,MyUW.net Email,,,,,,,no
-"Accounts, Identity, and Email",Email,all,https://uwnetid.washington.edu/manage/?forward,Email Forwarding,,,,,,,no
-"Accounts, Identity, and Email",Email,all,,,http://itconnect.uw.edu/connect/email,Email and Calendaring Options,https://www.uwb.edu/it/service-catalog/email-and-calendaring,Email and Calendaring Options,http://www.tacoma.uw.edu/admissions/setting-your-uw-email,Setting up your UW email,no
-"Accounts, Identity, and Email",Accounts and Identity,all,https://uwnetid.washington.edu/manage/,Manage UW NetID Account,,,,,,,no
-"Accounts, Identity, and Email",Accounts and Identity,all,http://services.uwmedicine.org/passwordportal,UW Medicine Password Portal,,,,,,,no
+"Email, and Accounts and Identity",Email,all,http://alpine.washington.edu/,Alpine Email,,,,,,,no
+"Email, and Accounts and Identity",Email,all,https://outlook.office365.com/uw.edu,UW Office 365 Email,,,,,,,no
+"Email, and Accounts and Identity",Email,all,http://mail.google.com/a/uw.edu,UW Gmail,,,,,,,no
+"Email, and Accounts and Identity",Email,all,http://outlook.com/owa/myuw.net,MyUW.net Email,,,,,,,no
+"Email, and Accounts and Identity",Email,all,https://uwnetid.washington.edu/manage/?forward,Email Forwarding,,,,,,,no
+"Email, and Accounts and Identity",Email,all,,,http://itconnect.uw.edu/connect/email,Email and Calendaring Options,https://www.uwb.edu/it/service-catalog/email-and-calendaring,Email and Calendaring Options,http://www.tacoma.uw.edu/admissions/setting-your-uw-email,Setting up your UW email,no
+"Email, and Accounts and Identity",Accounts and Identity,all,https://uwnetid.washington.edu/manage/,Manage UW NetID Account,,,,,,,no
+"Email, and Accounts and Identity",Accounts and Identity,all,http://services.uwmedicine.org/passwordportal,UW Medicine Password Portal,,,,,,,no
+"Email, and Accounts and Identity",Accounts and Identity,all,https://identity.uw.edu/,Identity.UW,,,,,,,
```

Comment:
It doesn't look like this link has a "no" or a "yes" to say whether it should be opened in a new tab or not.